schema.graphql

```diff
@@ -10,4 +10,18 @@ type FreeNFTDrop @entity(immutable: true) {
   description: String
   animationUrl: String
   imageUrl: String
+  
+  balances: [FreeNFTBalance!]! @derivedFrom(field: "drop")
+}
+
+
+type FreeNFTBalance @entity(immutable: false) {
+  "The owner is <drop address>-<owner address>"
```

Comment:
I don't understand this, do you mean this instead?

`"The id is <drop address>-<owner address>"`

Reply:
yes!